streampark-tests/streampark-testcontainer/src/main/java/org/apache/streampark/testcontainer/flink/FlinkStandaloneSessionCluster.java

```diff
@@ -45,19 +45,15 @@ public class FlinkStandaloneSessionCluster implements Startable {
 
   public static final Logger LOG = LoggerFactory.getLogger(FlinkStandaloneSessionCluster.class);
 
+  public static final int BLOB_SERVER_PORT = 6123;
```

Comment:
if these variables are only used within the current class, using private would be better.